core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2981,6 +2981,13 @@ public String toString() {
               + "an exponential backoff. This property determines the maximum duration to retry for"
               + " before giving up. Note that, this value is set to 5s for fs and fsadmin CLIs.")
           .build();
+  public static final PropertyKey USER_RPC_MAX_INBOUND_MESSAGE_SIZE =
+          new Builder(Name.USER_RPC_MAX_INBOUND_MESSAGE_SIZE)
+          .setDefaultValue("100MB")
```

Comment:
If the default value is 100MB, does this mean a 100MB buffer will always be allocated or only when the message is 100MB that the amount of memory will be allocated? I'm worried because our client is already a quite heavy on resources.

Reply:
The memory consumption is per use. This is more like a safety guard to prevent client from over loaded by unreasonably huge messages.

Reply:
This defines how big a message a client can accept from server. Client can create "up to" 100 MB with this change, not 100MB all the time.